protos/sigstore_trustroot.proto

```diff
@@ -81,10 +81,19 @@ message CertificateAuthority {
 // The embedded transparency logs, CT logs, CAs and TSAs MUST include any
 // previously used instance -- otherwise signatures made in the past cannot
 // be verified.
-// The currently used instances MUST NOT have their 'end' timestamp set in
-// their 'valid_for' attribute for easy identification.
+//
 // All the listed instances SHOULD be sorted by the 'valid_for' in ascending
-// order, that is, the oldest instance first and the current instance last.
+// order, that is, the oldest instance first. Only the last instance is
+// allowed to have their 'end' timestamp unset. All previous instances MUST
+// have a closed interval of validity. The last instance MAY have a closed
+// interval.
```

Comment:
> The last instance MAY have a closed interval.

What is the use case for this?

Reply:
I can't think of a good reason _I_ wanted to do so, but as an example, the current Fulcio root expires `Not After : Oct  5 13:56:58 2031 GMT` and I think it should be ok to have that value as `end`. 

And also this makes the language clear that clients must not treat the last instance with an open interval in any specific way.